homeassistant/components/sensor/switcher_kis.py

```diff
@@ -0,0 +1,264 @@
+"""Home Assistant Switcher Component.
+
+For controlling the Switcher Boiler Device (https://www.switcher.co.il/).
+Please follow configuring instructions.
+
+For use of type hints in comments for mypy checker:
+- Disabled pylint's 'W0611' (unused-import) warning.
+- Noqa'd flake8's 'F401' error (imported but unused).
+
+For pylint's incorrectly reports of no 'getLogger' in module 'logging':
+- Disabled pylint's 'E0611' (no-name-in-module) warning.
+
+Disabled pylint's 'C0326' (bad-whitespace).
+It contradicts flake8's 'E252'(missing whitespace around parameter equals).
+
+Author: Tomer Figenblat
+
+This cannot be configured as a sensor platform,
+Please follow the instruction of configuring the switcher_kis component.
+"""
+
+# pylint: disable=bad-whitespace,unused-import,no-name-in-module
+from asyncio import TimeoutError as AsyncioTimeoutError
+from asyncio import wait_for
+from logging import getLogger
+from typing import Any, Awaitable, Callable, Dict, List, Optional  # noqa F401
+
+from aioswitcher.consts import DISABLE_SCHEDULE, ENABLE_SCHEDULE
+from aioswitcher.devices import SwitcherV2Device  # noqa F401
+from aioswitcher.schedules import (SwitcherV2Schedule,
+                                   calc_next_run_for_schedule)
+from aioswitcher.swapi import delete_schedule, disable_enable_schedule
+
+from homeassistant.components.sensor import ENTITY_ID_FORMAT as SENSOR_FORMAT
+from homeassistant.components.switcher_kis import (DISCOVERY_CONFIG,
```

Comment:
Should be a relative import

Reply:
Thank you.
Fixed in [this](https://github.com/home-assistant/home-assistant/pull/20830/commits/6d478ac3f47207b344a670fc3a04f1811c544a83) commit.